app/src/main/java/com/andryoga/safebox/ui/view/login/LoginFragment.kt

```diff
@@ -44,6 +44,21 @@ class LoginFragment : Fragment() {
 
         setupObservers()
 
+        binding.ShowHintText.setOnClickListener {
+
+            if (binding.ShowHintText.text == getString(R.string.show_hint)) {
+
+                binding.ShowHintText.text = getString(R.string.hide_hint)
+                binding.displayHintText.visibility = View.VISIBLE
+                binding.hintDivider.visibility = View.VISIBLE
+            } else {
+
+                binding.ShowHintText.text = getString(R.string.show_hint)
```

Comment:
see if you can make a util function for this also... we can use this kind of method in other places as well.
function will be to switch one text to other. it can have two params : originalText, toChangeText.

Reply:
yeah sure I will try to implement that